src/settings/tabcpp.h

```diff
@@ -0,0 +1,50 @@
+#ifndef TABCPP_H
+#define TABCPP_H
+
+#include <QWidget>
+#include <QtWidgets>
+#include <QListWidget>
+#include <QComboBox>
```

Comment:
Header QtWidgets includes all elements of Qt UI
It is huge
You must remove this include from your files, especially - from headers
**_It might cause identifiers conflict_**

Reply:
removed